main.go

```diff
@@ -18,6 +20,101 @@ import (
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
+const (
+	diskPath             string = "./tmp/disk.img.gz"
+	diskPathDecompressed string = "./tmp/disk.img"
+	diskPathConverted    string = "./tmp/disk.qcow2"
+)
+
+func apply(vmName string) error {
+	log.Println("Applying VirtualMachineExport object...")
+
+	yaml := fmt.Sprintf(`
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+  name: %s
+spec:
+  source:
+    apiGroup: "kubevirt.io"
+    kind: VirtualMachine
+    name: %s
+`, vmName, vmName)
+	cmd := exec.Command("kubectl", "apply", "-f", "-")
+	cmd.Stdin = strings.NewReader(yaml)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func download(vmName, volumeName string) error {
+	log.Printf("Downloading disk image from %s Virtual Machine...\n", vmName)
+
+	cmd := exec.Command("usr/bin/virtctl", "vmexport", "download", vmName, "--vm", vmName, "--volume", volumeName, "--output", diskPath)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		return err
+	}
+
+	if fileInfo, err := os.Stat(diskPath); err != nil || fileInfo.Size() == 0 {
+		return fmt.Errorf("File does not exist or is empty.")
+	}
+
+	log.Println("Download completed successfully.")
+	return nil
+}
+
+func convert() error {
```

Comment:
rename it to something like `convertVMDiskToqcow`?

Reply:
Done.